collections/traitattr/occurattributes.php

```diff
@@ -49,360 +51,356 @@
+		} else {
+			echo '<a href="occurattributes.php?collid=' . htmlspecialchars($collid, HTML_SPECIAL_CHARS_FLAGS) . '&mode=1&traitid=' . htmlspecialchars($traitID, HTML_SPECIAL_CHARS_FLAGS) . '"><img src="../../images/edit.png" style="width:1.3em" />'.$LANG['EDIT'].'</a>';
 		}
-		?>
-		<div class="navpath">
-			<a href="../../index.php">Home</a> &gt;&gt;
-			<a href="../misc/collprofiles.php?collid=<?php echo htmlspecialchars($collid, HTML_SPECIAL_CHARS_FLAGS); ?>&emode=1">Collection Management</a> &gt;&gt;
-			<?php
-			if($mode == 2){
-				echo '<b>Attribute Reviewer</b>';
-			}
-			else{
-				echo '<b>Attribute Editor</b>';
-			}
-			?>
-		</div>
+		echo '</div>';
+	}
+	?>
+	<div class="navpath">
+		<a href="../../index.php"><?php echo $LANG['HOME'] ?></a> &gt;&gt;
+		<a href="../misc/collprofiles.php?collid=<?php echo htmlspecialchars($collid, HTML_SPECIAL_CHARS_FLAGS); ?>&emode=1"><?php echo $LANG['COLLECTION_MANAGEMENT'] ?></a> &gt;&gt;
 		<?php
-		if($statusStr){
-			echo '<div style="color:red">';
-			echo $statusStr;
-			echo '</div>';
+		if ($mode == 2) {
+			echo '<b>'.$LANG['ATTRIBUTE_REVIEWER'].'</b>';
+		} else {
+			echo '<b>'.$LANG['ATTRIBUTE_EDITOR'].'</b>';
 		}
 		?>
-		<!-- This is inner text! -->
-		<div id="innertext" style="position:relative;">
+	</div>
+	<?php
+	if ($statusStr) {
+		echo '<div style="color:red">';
+		echo $statusStr;
+		echo '</div>';
+	}
+	?>
+	<!-- This is inner text! -->
+	<div id="innertext" style="position:relative;">
 		<?php
-		if($collid){
-			?>
+		if ($collid) {
+		?>
 			<div style="float:right;width:290px;">
 				<?php
 				$attrNameArr = $attrManager->getTraitNames();
-				if($mode == 1){
-					?>
+				if ($mode == 1) {
+				?>
 					<fieldset style="margin-top:25px">
-						<legend><b>Filter</b></legend>
-						<form id="filterform" name="filterform" method="post" action="occurattributes.php" onsubmit="return verifyFilterForm(this)" >
+						<legend><b><?php echo $LANG['FILTER'] ?></b></legend>
+						<form id="filterform" name="filterform" method="post" action="occurattributes.php" onsubmit="return verifyFilterForm(this)">
 							<div>
 								<select name="traitid">
-									<option value="">Select Trait (required)</option>
+									<option value=""><?php echo $LANG['SELECT_TRAIT_REQ'] ?></option>
 									<option value="">------------------------------------</option>
 									<?php
-									if($attrNameArr){
-										foreach($attrNameArr as $ID => $aName){
-											echo '<option value="'.$ID.'" '.($traitID==$ID?'SELECTED':'').'>'.$aName.'</option>';
+									if ($attrNameArr) {
+										foreach ($attrNameArr as $ID => $aName) {
+											echo '<option value="' . $ID . '" ' . ($traitID == $ID ? 'SELECTED' : '') . '>' . $aName . '</option>';
 										}
-									}
-									else{
-										echo '<option value="0">No attributes are available</option>';
+									} else {
+										echo '<option value="0">'.$LANG['NO_ATTRI_AVAILABLE'].'</option>';
```

Comment:
Spaces before and after periods _are_ necessary within php fragments (I know, so specific!).